py_scripts/gif_generator.py

```diff
@@ -317,7 +317,11 @@ def plot_grouped_route_trace_results(grouped):
 # Run when this script is invoked
 if __name__ == '__main__':
     # First pull down the previous day's images
```

Comment:
TODO: Add a step to remove existing `busdata_raw/` directory and make a new one (so, clear out the existing one).